spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkMicroBatchStream.java

```diff
@@ -200,10 +201,13 @@ public void stop() {
   private boolean shouldProcess(Snapshot snapshot) {
     String op = snapshot.operation();
     Preconditions.checkState(!op.equals(DataOperations.DELETE) || skipDelete,
-        "Cannot process delete snapshot: %s", snapshot.snapshotId());
+        "Cannot process delete snapshot: %s, to ignore snapshots of type delete, set the config %s to true.",
```

Comment:
I think this should be `"to ignore deletes, set %s=true"`.

Reply:
done